app/lib/presentation/pages/home_page.dart

```diff
@@ -5,6 +5,8 @@ import 'package:word_link/presentation/core/global_variables.dart';
 import 'package:word_link/presentation/molecules/molecules.dart';
 import 'package:word_link/presentation/organisms/detailed_card_organism.dart';
 import 'package:word_link/presentation/pages/custom_collections_page.dart';
+import 'package:word_link/presentation/pages/pages.dart';
+import 'package:word_link/domain/controllers/words_controller.dart';
```

Comment:
We are not importing individual controllers, add `words_controller.dart `to `controllers.dart `